nni/compression/experiment/experiment.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+from pathlib import Path
+from typing import Any, Callable, List, Optional, Union
+
+import torch
+from torch.nn import Module
+from torch.optim import Optimizer
+
+from nni.compression.experiment.config.utils import cc_cv2ss
+from nni.experiment import Experiment, ExperimentConfig
+from .config import CompressionConfig, CompressionVessel
+
+_logger = logging.getLogger('nni.experiment')
+
+
+class CompressionExperiment(Experiment):
+    def __init__(self, config_or_platform: Union[ExperimentConfig, str, List[str]],
+                 compression_config: CompressionConfig, model: Module, finetuner: Callable[[Module], None],
```

Comment:
Please remember to revise whether to put additional args into experiment config before release.
c.c. @QuanluZhang